### ISSUE TITLE
in-body unsubscribe link not found in text-only messages?

### ISSUE THREAD

**Issue by mjl-** [wttw/aboutmyemail#38]
I sent a message with a 1-click unsubscribe link, and the same link in the message. It is a text-only message. I initially had the link in the footer, below the `--`, moving it up did not make a difference.
Perhaps the checker only looks for links in text/html parts? I would expect a URL in a text-only message to be fine, mail clients will make it clickable.

For https://aboutmy.email/5715f57/yahoogle.

Btw, I tried looking in the code for the checks, but couldn't find them. Are they around? Hope I'm in the right repo...

**Comment by wttw** [comment]
Yes, we only look in the HTML part for in-body unsubscription links, so they'll never be found in pure text/plain messages.

If you're the rare sender that's sending marketing bulk mail as text/plain then you know better than this tool. :)

More support for text/plain would be something to add, but it's not high up the priority list at the moment.

The app isn't open source (although I've released some of the code it's based on elsewhere); this repo contains some fragments where it's useful to make them publicly available, and also contains the source for the client apps that can be used to access the API.
